browser/resources/resource_ids

```diff
@@ -121,4 +121,7 @@
   "<(ROOT_GEN_DIR)/brave/web-ui-brave_vpn_panel/brave_vpn_panel.grd": {
     "includes": [51000]
   },
+  "<(ROOT_GEN_DIR)/brave/web-ui-trezor_bridge/trezor_bridge.grd": {
```

Comment:
We need to stop adding grd files like this. Please create `brave/browser/browser_resources.grd` and make `trezor_bridge` a grdp file

Reply:
That's going to be beyond the scope of this PR. The webpack GN template spits out a GRD, so that would need to be changed to GRDP. That's going to affect _a lot_ of things since we depend on a specific `kMyGRDFileResourceMap` being created for each UI, and not 1 map of every resource for the whole browser.

Reply:
ok, I didn't realize this was part of a larger process and couldn't easily be changed